scripts/irssi_logger.pl

```diff
@@ -90,29 +93,35 @@ sub connect_db {
     my $dbname = Irssi::settings_get_str('il_dbname') || $user;
     my $dbuser = Irssi::settings_get_str('il_dbuser') || $user;
     my $dbpass = Irssi::settings_get_str('il_dbpass') || "";
+    my $dbhost = Irssi::settings_get_str('il_host');
+    my $dbport = Irssi::settings_get_str('il_port');
 
     db_init($dbname, $dbuser, $dbpass);
 
     Irssi::print("Connecting to the database");
 
-    return DBI->connect("dbi:Pg:dbname=$dbname", $dbuser, $dbpass) || Irssi::print("Can't connect to db!" . DBI::errstr);
+    return DBI->connect("DBI:Pg:database=$dbname;host=$dbhost;port=$dbport", $dbuser, $dbpass) || Irssi::print("Can't connect to db!" . DBI::errstr);
 }
 
 sub write_db {
     my ($nick, $message, $target) = @_;
-    my @vals;
-    my $date = strftime("%Y-%m-%d %H:%M:%S", localtime);
+    if (exists $blacklist{$target}) {
```

Comment:
Personally, i'd just do an `if (!exists` here and exclude the `} else {`.

Reply:
sure